packages/slice-machine/tests/components/UpdatePromptAndDesktop.spec.tsx

```diff
@@ -62,13 +66,14 @@ const FAKE_ENVIRONMENT = {
     framework: Framework.next,
     baseUrl: "http://localhost:9999",
     client: new FakeClient(),
-  },
+    tracker: new Tracker("fake tracker", "test-version"),
+  } as Environment,
 };
 
 const STATE_FOR_UPDATE = {
   environment: FAKE_ENVIRONMENT,
 };
-test("when not upto date it should open a model and provided update instructions", async () => {
+test.skip("when not up to date it should open a model and provided update instructions", async () => {
```

Comment:
Was skipped because of the modal opening to many times If I recall correctly.
Should be kept in mind when the PR is fixed.